packages/core/helper-plugin/src/features/CustomFields.tsx

```diff
@@ -85,12 +85,14 @@ interface CustomField {
   options?: CustomFieldOptions;
 }
 
+type CustomFieldUID = `plugin::${string}.${string}` | `global::${string}`;
+
 /* -------------------------------------------------------------------------------------------------
  * Context
  * -----------------------------------------------------------------------------------------------*/
 
 interface CustomFieldsContextValue {
-  get: (uid: string) => CustomField | undefined;
+  get: (uid: CustomFieldUID) => CustomField | undefined;
```

Comment:
I wonder if this should be typed in strapi types instead. Don't have the time to look at it now, but we could take a look because we do have some uid's definitions in there

Reply:
That could be nice, it's a good "lil" task during refactor.. ideally we really need to document that package etc. even if it's with API Extractor because it's a closed-box and you only know, if you know 🤓